### ISSUE TITLE
[Bug] fp8 measure failed on Llama-2-7b-hf

### ISSUE THREAD

**Issue by ClarkChin08** [huggingface/optimum-habana#890]
### System Info

```shell
Optimum habana main branch with commit :fec2e509ae3aecef61f52037d8498e3315b79a44
habana_quantization_toolkit version: 1.15.1.15
Torch version：2.1.0a0+gitf8b6084
```

### Information

- [X] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

1. Run measure in examples/text-generation:
QUANT_CONFIG=quantization_config/maxabs_measure.json   python run_generation.py \
      --use_hpu_graphs \
      --use_kv_cache \
      --limit_hpu_graphs \
      --trim_logits \
      --bf16 \
      --batch_size 1 \
      --fp8 \
      --model_name_or_path ~/models--meta-llama--Llama-2-7b-hf/

### Expected behavior

![image](https://github.com/huggingface/optimum-habana/assets/52476578/b282bf2c-b6c0-4a05-b6d3-54390eaf3d2c)

**Comment by libinta** [comment]
@ClarkChin08 measurement should be done in bf16, so --fp8 needs to be removed. the run_generation.py will generate the measurement based on few data sample, it's not good for accuracy in real use case. please use  run_lm_eval.py. also there are few arguments are missing. please follow the example from https://github.com/huggingface/optimum-habana/tree/main/examples/text-generation#running-with-fp8 section for measurement.
QUANT_CONFIG=./quantization_config/maxabs_measure.json python run_lm_eval.py \
-o acc_70b_bs1_measure.txt \
--model_name_or_path meta-llama/Llama-2-7b-hf \
--attn_softmax_bf16 \
--use_hpu_graphs \
--trim_logits \
--use_kv_cache \
--reuse_cache \
--bf16 \
--batch_size 1

Thanks.
